### PR TITLE
Add Mopidy 3 FAQ post

### PR DESCRIPTION
**Pull request by kingosticks**

### CODE REVIEW

**Comment by kingosticks** [comment]
This shouldn't be merged until the debs are done, and i still need to fix a bit of the Ubuntu part.

**Comment by kingosticks** [comment]
Ugh, that 18.04 section is nasty. Grabbing that package is horrible and since it's not pinned it'll presumably get upgraded on the next `apt update`.

File: _posts/2019-12-27-mopidy-3-faq.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+Python 3.6. The following steps are not pretty, but should get you up and
+running.
+
+First up is pygobject: the Python 3 bindings for gobject-introspection
+(`python3-gi` package). This is easy to build from source using pip once the
+required dependencies have been installed:
+
+```
+sudo apt install -y \
+    python3-gst-1.0 \
```
Comment (jodal) [inline _posts/2019-12-27-mopidy-3-faq.md]: Maybe sort this package list?

**Comment by kingosticks** [comment]
I was thinking of trying an alternate method using deb-src (pointing to cosmic) and building with apt might be cleaner. Otherwise definitely needs something like `apt-mark hold` to stop that random package being upgraded away.

**Comment by kingosticks** [comment]
OK, I think that is much better. This way we get all the correct Mopidy 3 service stuff.

**Pull request merged**